mep3_simulation/webots_data/worlds/test_opponent_robot.wbt

```diff
@@ -0,0 +1,651 @@
+#VRML_SIM R2022a utf8
+WorldInfo {
+  basicTimeStep 10
+}
+Viewpoint {
+  orientation -0.21115345114736453 0.21434687898294047 0.9536611743899559 1.6032403587512039
+  position -0.10785614864637297 -1.293760151914316 0.6340978885137727
+}
+TexturedBackground {
+}
+TexturedBackgroundLight {
+}
+Floor {
+  name "table"
+  size 3 2
+  tileSize 4.096 2.048
+  appearance PBRAppearance {
+    baseColorMap ImageTexture {
+      url [
+        "assets/table.png"
+      ]
+      repeatS FALSE
+      repeatT FALSE
+    }
+    roughness 1
+    metalness 0
+  }
+}
+Solid {
```

Comment:
Treba nam samo sto, ostali detalji (pakovi, galerija...) nam ne trebaju u testu

Reply:
Ostavila sam zidove koji se nalaze oko stola. Da li i to da obrisem?

Reply:
Mozes i njih obrisati, nisu relevantni za test